src/index.js

```diff
@@ -15,12 +15,107 @@ const unexpectedEnzyme = {
       identify: function(reactWrapper) {
         return reactWrapper && reactWrapper instanceof ReactWrapper;
       },
-      inspect: function(reactWrapper, depth, output) {
-        const inspected = reactWrapper.exists()
-          ? reactWrapper.getElement()
-          : reactWrapper.root().html();
+      inspect: function(reactWrapper, depth, output, inspect) {
+        if (!reactWrapper.exists()) {
+          return output.jsKeyword('null');
+        }
+
+        if (reactWrapper.length > 1) {
+          return output.appendItems(reactWrapper.map(node => node), '\n');
+        }
+
+        const startTag = output.clone();
+        startTag.text('<');
+        startTag.jsKeyword(reactWrapper.name());
+
+        const props = reactWrapper.props();
+        Object.keys(props).forEach(key => {
+          if (key === 'children') {
```

Comment:
Would it make sense to check for `children` to also be of type `string|node`? Maybe we want to do something different for render props?

Reply:
Don't we always want to render the childrens inside the tag, this is just preventing us from inspecting them as a prop.

Reply:
Let's see what happens, it is a little bit hard to predict all problems that can arise. We need some more tests and real world usage.